ktx-library-codelab/step-06/myktxlibrary/src/main/java/com/example/android/myktxlibrary/LocationUtils.kt

```diff
@@ -58,7 +58,7 @@ fun FusedLocationProviderClient.locationFlow() = callbackFlow<Location> {
             result ?: return
             for (location in result.locations) {
                 try {
-                    offer(location) // emit location into the Flow using ProducerScope.offer
+                    trySend(location).isSuccess // emit location into the Flow using ProducerScope.offer
```

Comment:
`isSuccess` doesn't do anything here (the result is dropped anyway), so maybe we could remove it?

Reply:
There was an error message: Using 'offer(E): Boolean' is an error. Deprecated in the favour of 'trySend' method. It has proven itself as the most error-prone method in Channel API.

Reply:
I agree with the change to trySend(location), I was only asking to remove isSuccess

Reply:
Sorry for misunderstanding.

Reply:
iSuccess removed.